### PR TITLE
Allow user to revoke invitation created by itself

### PR DESCRIPTION
**Pull request by chenhunghan**

### CODE REVIEW

**Review by panuhorsmalahti — APPROVED** [review]

File: src/Permissions.ts [hunk 1]
```diff
@@ -49,7 +59,25 @@ export class Permissions {
         canI = this.getRole(forSpace, forUserId) !== Roles.None;
         break;
       case Actions.PatchInvitation:
-      case Actions.RevokeInvitation:
+      case Actions.RevokeInvitation: {
```
[the remark this answers is not in the document]
Comment (chenhunghan) [reply src/Permissions.ts:62]: that will remove `RevokeInvitation` in `canSpace` => breaking change, right? if so I suggest maybe later

File: src/Permissions.ts [hunk 2]
```diff
@@ -35,10 +36,19 @@ export class Permissions {
...
    * @throws "Could not get role for space with no teams" exception
    */
-  canSpace(action: Actions, forSpace: Space, forUserId: string) {
+  canSpace(
+    action: Actions,
+    forSpace: Space,
+    forUserId: string,
+    forRevokeInvitation?: {
+      invitationId: string;
+      invitationsCreatedByUserId: Invitation[];
```
Comment (panuhorsmalahti) [inline src/Permissions.ts:49]: Why don't we just require "createdById" instead of all invitations?

Comment (chenhunghan) [reply src/Permissions.ts:49]: that's true...

File: src/Permissions.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (panuhorsmalahti) [reply src/Permissions.ts:62]: Yes, breaking change.. I was planning to do this before but forgot. Now we keep adding parameters that are not really related to Spaces. But we can merge this and fix later.

File: src/Permissions.ts (hunk 2 above)
Comment (chenhunghan) [reply src/Permissions.ts]: Updated https://github.com/lensapp/lens-platform-sdk/pull/85/commits/b2dcc07c974d7049508a46a1564bfc7c8a16fcb1

**Pull request merged**